### DISCUSSION TITLE
org community discussions api

### DISCUSSION THREAD

**Discussion by kazupot7** [community/community#124067] [Programming Help]
### Body

I need api how to use it

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebndev** [comment]
Hi @kazupot7,

You can find the documentation for the Discussions API in [this previous discussion](https://github.com/orgs/community/discussions/23483#discussioncomment-3240554)!
